src/unfccc_ghg_data/unfccc_reader/Mongolia/read_MNG_BUR2_from_pdf.py

```diff
@@ -302,24 +523,36 @@ def repl(m):  # noqa: D103
         encoding=encoding,
     )
 
-    # ###
-    # Processing
-    # ###
+    # # ###
+    # # Processing
+    # # ###
 
-    data_proc_pm2 = process_data_for_country(
+    # create the gas baskets before aggregating the categories
+    data_proc_pm2_gas_baskets = process_data_for_country(
         data_country=data_pm2,
         entities_to_ignore=[],
         gas_baskets=gas_baskets,
         filter_dims=None,
         cat_terminology_out=None,
         category_conversion=None,
         sectors_out=None,
+        processing_info_country=None,
+    )
+
+    data_proc_pm2 = process_data_for_country(
+        data_country=data_proc_pm2_gas_baskets,
+        entities_to_ignore=[],
+        gas_baskets=None,
+        filter_dims=None,
+        cat_terminology_out=None,
+        category_conversion=None,
+        sectors_out=None,
         processing_info_country=country_processing_step1,
```

Comment:
Are all sectors and gases covered for all years, or do you se downscaling or do we have gaps? If there are gaps in subsectors / gases and no downscaling yet, we could take a look together.